src/overlays/actors/ovl_En_Gm/z_en_gm.c

```diff
@@ -7,8 +14,27 @@
 void EnGm_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnGm_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnGm_Update(Actor* thisx, GlobalContext* globalCtx);
+void EnGm_DrawProc(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
```suggestion
void EnGm_Draw(Actor* thisx, GlobalContext* globalCtx);
```
even if draw is set in code, like `BgHakaMegane_Draw`